app/controllers/hosts_controller.rb

```diff
@@ -48,18 +48,20 @@ def index(title = nil)
       error e.to_s
       search = resource_base.search_for ''
     end
+    @hosts = search.includes(included_associations).paginate(:page => params[:page])
```

Comment:
testing with newer version is a lot faster:
```
17:12:27 rails.1   | 2017-02-12T17:12:27 c58ea131 [app] [I] Sent data hosts-2017-02-12.csv (0.5ms)
17:12:27 rails.1   | 2017-02-12T17:12:27 c58ea131 [app] [I] Completed 200 OK in 61069ms (Views: 0.4ms | ActiveRecord: 1690.6ms)
```

However, most of the AR time was spent in Token and Host Status loading, both are not reflected in the CSV, in that regard, it does open the question if we should enable power status and host-status visibility or not in the csv.